Frends.AmazonS3.ListObjects/Frends.AmazonS3.ListObjects/Frends.AmazonS3.ListObjects.csproj

```diff
@@ -0,0 +1,30 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+	<TargetFrameworks>net6.0</TargetFrameworks>
+	<Version>1.0.0</Version>
+	<Authors>Frends /ttossavainen</Authors>
```

Comment:
Author is just Frends.

Reply:
Removed. Jefim asked me to add myself as author on another repo.